RecoMuon/DetLayers/plugins/MuonDetLayerGeometryESProducer.cc

```diff
@@ -12,11 +13,13 @@
 #include <Geometry/CSCGeometry/interface/CSCGeometry.h>
 #include <Geometry/RPCGeometry/interface/RPCGeometry.h>
 #include <Geometry/GEMGeometry/interface/GEMGeometry.h>
+#include <Geometry/GEMGeometry/interface/ME0Geometry.h>
```

Comment:
I mean all of the includes of CMSSW package files
It's mostly style, not required